Makefile

```diff
@@ -365,6 +365,12 @@ pyflakes: reports
 pylint: reports
 	@(set -o pipefail && $@ | reports/$@.report)
 
+swagger: reports
+	@if [ "$(VENV_BASE)" ]; then \
+		. $(VENV_BASE)/awx/bin/activate; \
+	fi; \
+	(set -o pipefail && py.test awx/main/tests/docs --release=$(RELEASE_VERSION) | tee reports/$@.report)
```

Comment:
The docs are built via a functional test that:

1.  Does a `GET` to a new `settings.DEBUG`-only Swagger view (powered by the `django-rest-swagger` library).
2. Does various transformations.
3. Writes a JSON file.

I'd also like to add _actual_ tests that does some sanity checks on the generated output so we know if dependency or view/serializer changes have broken something.